scripts/build.ts

```diff
@@ -3,16 +3,23 @@ import fs from 'fs';
 import stringify from 'fast-json-stable-stringify';
 
 import features from '../index.js';
+import { execSync } from 'child_process';
 
 const rootDir = new URL('..', import.meta.url);
 const packageDir = new URL('./packages/web-features/', rootDir);
 
+const filesToCopy = ["LICENSE.txt", "types.ts"];
+
 function build() {
     const json = stringify(features);
     // TODO: Validate the resulting JSON against a schema.
     const path = new URL('index.json', packageDir);
     fs.writeFileSync(path, json);
-    fs.copyFileSync(new URL("LICENSE.txt", rootDir), new URL("LICENSE.txt", packageDir));
+    for (const file of filesToCopy) {
+        fs.copyFileSync(new URL(file, rootDir), new URL(file, packageDir));
+    }
+    execSync("npm install", { cwd: "./packages/web-features"});
+    execSync("npm run prepare", { cwd: "./packages/web-features"});
```

Comment:
Here I add the steps to generate the types. When this is done, all of the files needed for `npm pack` or `npm publish` to work are present in `packages/web-features`